packages/sdk-metrics/src/aggregator/ExponentialHistogram.ts

```diff
@@ -0,0 +1,606 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Accumulation,
+  AccumulationRecord,
+  Aggregator,
+  AggregatorKind,
+} from './types';
+import {
+  DataPointType,
+  ExponentialHistogramMetricData,
+} from '../export/MetricData';
+import { diag, HrTime } from '@opentelemetry/api';
+import { InstrumentDescriptor, InstrumentType } from '../InstrumentDescriptor';
+import { Maybe } from '../utils';
+import { AggregationTemporality } from '../export/AggregationTemporality';
+import { Buckets } from './exponential-histogram/Buckets';
+import { getMapping } from './exponential-histogram/mapping/getMapping';
+import { Mapping } from './exponential-histogram/mapping/types';
+import * as util from './exponential-histogram//util';
```

Comment:
I usually avoid importing such a generic name like util in favor of named imports